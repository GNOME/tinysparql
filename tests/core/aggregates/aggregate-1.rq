PREFIX : <http://example/> 

SELECT SUM(?v) AVG(?w) MAX(?v) MIN(?v)
{ ?_x :i1 ?v ;
      :i2 ?w }
