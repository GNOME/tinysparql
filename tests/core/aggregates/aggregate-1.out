"9"	"3.33333333333333"	"5"	"1"
